src/edu/csus/ecs/pc2/AppConstants.java

```diff
@@ -162,4 +162,10 @@ public class AppConstants {
      */
     public static final String DATETIME_PATTERN_LONG = "E, dd MMM yyyy HH:mm:ss z";
     
+    
+    /**
+     * The value is a full path to a event-feed.json file.
+     * 
+     */
+    public static final String OVERRIDE_EVENT_FEED_FILE = "server.eventFeedFile";
```

Comment:
The comment says "the value is a full path", but the value doesn't appear to be a full path...
I suggest either correcting the comment, or providing a full path in the value.

Reply:
will update comment, note there is no check or control over whether uses a full path, partial path or no path,.

Reply:
pushed fix changed comment to * The value is the name of an event-feed.json file.